tools/ci_build/github/azure-pipelines/linux-cpu-minimal-build-ci-pipeline.yml

```diff
@@ -245,6 +252,70 @@ jobs:
               --cmake_extra_defines onnxruntime_ENABLE_ORT_FORMAT_RUNTIME_GRAPH_OPTIMIZATION=ON
       workingDirectory: $(Build.SourcesDirectory)
 
+  - script: git checkout -- .
```

Comment:
this could probably move between 7a and 7b (possibly also after 7b in case we ever add more steps). steps 6a and 6b shouldn't update the code with any op exclusions.

Reply:
Doing it this way is a little more consistent as it matches the pattern of doing a cleanup, even if not strictly necessary, between each main step. 

This way you only have to think about additional cleanup needs within a step (i.e. between the 'a' and 'b') and don't require knowledge across steps. I shouldn't need to know everything that happened within 6a and 6b to decide if cleanup is necessary or not before step 7. 

Not needed between 7a and 7b as they're both disabling all ops. 

Adding it after feels unnecessary as we may never add another step.

Reply:
my interpretation of the pattern was that each numbered step (including 'a', 'b') that modifies the source cleans up after itself before the next numbered step.
I think this works though